src/App/components/Sheet/index.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Icon } from "#";
+
+const Sheet = ({ id, requireAction, children }) => (
+    <div className="sheet" id={id} data-require-action={requireAction}>
```

Comment:
I think that in the documentation, if someone adds `<Sheet requireAction={false} />` it will render as `<div class="sheet" data-require-action="false">...</div>` which again will make the script ignore the sheet, even thoguh it is not intended. Im not 100% sure on this, but it might be worth a check.

```suggestion
    <div className="sheet" id={id} data-require-action={requireAction ? true : null}>
```